core/modules/file/js/file.js

```diff
@@ -95,14 +85,18 @@ Backdrop.file = Backdrop.file || {
         event.filePreValidation = false;
         return false;
       }
+      else {
+        event.filePreValidation = true;
+      }
     }
   },
   /**
    * Automatically upload files by clicking the Upload button on file selection.
    */
   autoUpload: function (event) {
     // This value is set in Backdrop.file.validateExtension().
-    if (event.filePreValidation) {
+    if (event.filePreValidation === undefined || event.filePreValidation === true) {
+      consoe.log('autoupload');
```

Comment:
Nevermind, fixed in https://github.com/backdrop/backdrop/commit/e6c5075880f5f76e6a12fc14cf05335f54ce0eed